src/ruler.js

```diff
@@ -22,12 +22,24 @@ export class DragRulerRuler extends Ruler {
 		// This function is invoked by left clicking
 		if (!this.isDragRuler)
 			return await super.moveToken(event);
+
+		let options = {};
+
+		// Allow outside modules to override snapping
+		if (this.snapOverride !== undefined && this.snapOverride.active) {
```

Comment:
This is the same code as in `main.js`, isn't it? Let's put that into a function to avoid duplicated code.

Reply:
Found a typo bug where I was setting 'snapOverrideData' inside of my module instead of 'snapOverride'. Fixed that in v2.4.3